pkg/reconciler/shared/tektonconfig/upgrade/upgrade.go

```diff
@@ -32,7 +32,6 @@ import (
 var (
 	// pre upgrade functions
 	preUpgradeFunctions = []upgradeFunc{
-		upgradeChainProperties,      // upgrade #1: upgrade chain properties
 		resetTektonConfigConditions, // upgrade #2: removes conditions from TektonConfig CR, clears outdated conditions
```

Comment:
@PuneetPunamiya can you please reindex the text of the second upgrade func?
 `// upgrade #2` -> `// upgrade #1`

Reply:
Done